skycoin/skywire
# Use retrier from the dmsg package netutil

Did you run `make format && make check`?
yes

Fixes #1040 

 Changes:	
- Changed retrier to dmsgnetutil 
- Removed internal netutil retrier
- Removed warn log from app discovery

How to test this PR:
1. Checkout [#dmsg-162](https://github.com/skycoin/dmsg/pull/162)
2. Uncomment `//replace github.com/skycoin/dmsg => ../dmsg` in go mod an run `make dep; make build`
3. Create config `./skywire-cli config gen -dir`
4. Run visor `./skywire-visor -c skywire-config.json`
5. See retrier warn logs
6. Close visor with `Ctrl+C` and see if there are no retrier warn logs while closing

Needs:
[#dmsg-162](https://github.com/skycoin/dmsg/pull/162)
